tofhir-server/src/main/scala/io/tofhir/server/endpoint/CodeSystemEndpoint.scala

```diff
@@ -24,7 +25,15 @@ class CodeSystemEndpoint(codeSystemRepository: ICodeSystemRepository) extends La
       } ~ pathPrefix(Segment) { codeSystemId =>
         pathEndOrSingleSlash {
           getCodeSystemRoute(terminologyId, codeSystemId)
-        } ~ pathPrefix(SEGMENT_CONTENT) {
+        } ~ pathPrefix(SEGMENT_CONTENT) { // code-systems/<code-system-id>/content
```

Comment:
Can you please add a comment/description to clarify the difference between content, header and file (maybe with some short examples)?